transcom/mymove
# INT B-18758 HL submit ppm doc

## [B-18758](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18758)

## Summary

Quick change that changes the title and status message of the event that is logged when the user submits their PPM documentation and  sends the move to the PPM closeout queue.

### How to test

1. Login as user and create a move, create a PPM shipment, and sign/send to SC.
2. Approve the move details as an SC.
3. Complete the PPM documentation upload process as the user.
4. Go back to SC (or any gov user), find move in the search tab, go to Move History tab.
5. Near top you should see an event log with the new title and status message

![Screenshot 2024-03-29 at 12 29 21 PM](https://github.com/transcom/mymove/assets/146959895/2b6749dc-4f42-4828-86da-56ef09472671)
